src/main/java/labs/introtoprogramming/lab5/object/Builder/POJO/FacePlate.java

```diff
@@ -0,0 +1,54 @@
+package labs.introtoprogramming.lab5.object.Builder.POJO;
+
+import java.util.ArrayList;
+
+public class FacePlate {
+
+    public ArrayList<VertexFace> vertices = new ArrayList<VertexFace>();
+    public Material material = null;
+    public Material map = null;
+
+    public FacePlate() {
+    }
+
+    public void add(VertexFace vertex) {
+        vertices.add(vertex);
+    }
+    public VertexNormal faceNormal = new VertexNormal(0, 0, 0);
```

Comment:
Make it private or at least package-private. Normal calculation can be put off for some time.